lib/python/rose/apps/comparisons/within.py

```diff
@@ -75,9 +81,16 @@ def __init__(self, task, val1, val2):
           self.percentage = "XX"
 
     def __repr__(self):
-        return OUTPUT_STRING % ( self.extract, self.percentage, FAIL,
-                                 self.tolerance, self.resultfile,
-                                 self.kgo1file,  self.numvals )
+        if self.numvals == 1:
+            return OUTPUT_STRING_WITH_VALS % ( self.extract, self.percentage, 
```

Comment:
Extra space after open bracket breaks PEP8. Ditto for space before close bracket.